lib/theme_check/file_system_storage.rb

```diff
@@ -12,7 +12,7 @@ def initialize(root, ignored_patterns: [])
     end
 
     def path(relative_path)
-      @root.join(relative_path)
```

Comment:
Is this really needed? Would prevent using any `Pathname` methods on this later.

Reply:
Actually I did it for consistency with the other one and because I was using + in the unit tests. I definitely can change both to paths!